src/features/commands/character/character.ts

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash'
+import { Message } from 'discord.js'
+import { CharacterService } from '../../../services/character/character.service'
+import { MessageManagerService } from '../../../services/message-manager/message-manager.service'
+
+export class CharacterCommand {
+  private static _instance: CharacterCommand
+
+  public static getInstance(): CharacterCommand {
+    if(_.isNil(CharacterCommand._instance)) {
+      CharacterCommand._instance = new CharacterCommand()
+    }
+    return CharacterCommand._instance
+  }
+
+  public displayCharacterMessage(message: Message) {
```

Comment:
Since you are already in the character command class you could just sum up the method name to message.
public message(...)

Reply:
Will do!